README.md

```diff
@@ -1,6 +1,16 @@
 Chitter Challenge
 =================
 
+SO FAR
+-------
+
+First 3 user stories working
+All tests passing and test database set up
+Would like to make Time be determined by system time. How?
+Need to allow user to have unique id to log in. New Table?
```

Comment:
Yes - then you can use a 'foreign key' relationship to make sure the peep is attached to a user.